gnovm/cmd/gno/bug.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"net/url"
+	"os/exec"
+	"runtime"
+	"strings"
+	"text/template"
+
+	"github.com/gnolang/gno/tm2/pkg/commands"
+)
+
+const bugTmpl = `## [Subject of the issue]
```

Comment:
```suggestion
// NOTE: keep in sync with .github/ISSUE_TEMPLATE/BUG-REPORT.md
const bugTmpl = `## [Subject of the issue]
```

Reply:
+ modify the file, place a note in the front matter (using a yaml comment)

```yaml
# NOTE: keep in sync with gnovm/cmd/gno/bug.go
```

Reply:
Should be resolved by: https://github.com/gnolang/gno/pull/1325/commits/5e655b8e9637815987eba0ffbca7be6250af7dfa